src/state/services/user-service.js

```diff
@@ -0,0 +1,97 @@
+import client from './supabase-client';
+
+//gets the user from supabase
```

Comment:
these comments are going to be super helpful when you look back at this in a month or so, good work